packages/crew_profile.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Crew_profile < Package
+  description 'A profile for Chromebrew, to be used instead of ~/.*rc'
+  homepage 'https://github.com/skycocker/chromebrew'
+  version '0.4.3'
+  source_url 'https://raw.githubusercontent.com/skycocker/chromebrew/master/install.sh'
```

Comment:
This is a moving target.  As soon as your PR is merged into master, the sha256 of this will be different.  If you want to have something more static, use a hashed version of a file like `https://github.com/skycocker/chromebrew/blob/511ee9cb3138e113df74ddf0dee057ef5d9a46fd/README.md`.

Reply:
Even if you change the sha256 hash, this is still a moving target.  Any new commits to install.sh on the master branch is going to break this.

Reply:
Got it. Will change.